docs/source/concepts/settlement.rst

```diff
@@ -7,38 +7,147 @@ Settlement
+
+.. image:: ../images/settlement_allocated.png
+   :alt: Settlement is allocated.
+
+Each receiver can then specify to which account the holding should be sent to by ``approving`` the corresponding instruction.
+
+.. image:: ../images/settlement_allocated_approved.png
+   :alt: Settlement is allocated and approved.
+
+Execute
+=======
+
+Once instructions are allocated and approved, a Settler party uses the ``Batch`` contract to ``execute`` them and finalize settlement in one atomic transaction.
+
+.. image:: ../images/settlement_executed.png
+   :alt: Settlement is allocated and approved.
+
+The instructions and batch are archived following a successful execution.
+
+Remarks
+=======
+
+There are some assumptions that need to be verified in order for the transaction above to settle successfully in practice. On the first instruction
+
+- Bob needs to have an account at the custodian where Alice's holding is held
+- Alice's holding needs to be :ref:`Transferable <type-daml-finance-interface-holding-transferable-transferable-24986>`
+- The transfer must be fully authorized (TODO : add a link to a page where we describe the role of account instructors and approvers, probably somewhere in the asset model)
+
+Also, note that the allocation and approval steps can happen in any order.
+
+The components in detail
+************************
+
+Settlement factory
+==================
+
+The :ref:`Settlement Factory <type-daml-finance-interface-settlement-factory-factory-31525>` is used to instruct settlement, i.e., create the ``Batch`` contract and the settlement ``Instructions``.
+
+The role of the Settlement Factory becomes important when a transfer requires intermediaries to be involved.
+Let us assume, for instance, that Alice's EUR holding in the example above is held at Bank A, whereas Bob has a EUR account at Bank B. Bank A and Bank B both have accounts at the Central Bank.
+
+.. image:: ../images/settlement_hierarchy.png
+   :alt: Hierarchical account structure. Alice has an account at Bank A. Bob has an account at Bank B. Bank A and Bank B have an account at the Central Bank.
+
+In this case, a direct holding transfer from Alice and Bob cannot generally be instructed. The original step needs to be broken down by the Settlement Factory into three separate ``Instructions``:
+
+- **1A**: Alice sends 1000 EUR (held at Bank A) back to Bank A
+- **1B**: Bank A sends 1000 EUR (held at the Central Bank) to Bank B.
+- **1C**: Bank B credits 1000 EUR to Bob's account (held at Bank B)
+
+.. image:: ../images/settlement_hierarchy_instructed.png
+   :alt: Instructions for intermediated settlement: Alice sends 1000 EUR to Bank A. Bank A sends 1000 EUR to Bank B. Bank B sends 1000 EUR to Bob.
+
+We refer to this scenario as *settlement with intermediaries*, or just *intermediated settlement*.
+The Settlement Factory is responsible for generating the correct set of instructions, such that full allocation and approval are possible.
```

Comment:
What does "full allocation and approval" mean in practice?

Reply:
Ok, I see it in the table below. Maybe add something like "full allocation and approval, as described in the table below"?

Reply:
What I meant to say is that the instructions can effectively be allocated and approved. This would not be possible if for instance we have an instruction A -> B but these two parties do not have an account at a shared custodian.

Reply:
Let me know if you like the updated wording and thanks for taking the time to review!

Reply:
I would maybe prefer a formulation like "so that they can be allocated and approved by the respective parties" (if that is a correct description).

Reply:
Cool, I like it